### DISCUSSION TITLE
Github Discussion theme design ideas

### DISCUSSION THREAD

**Discussion by volkantash** [community/community#72834] [Discussions]
### Select Topic Area

Show & Tell

### Body

Selam. Tasarımın böyle olmasını ister misiniz? Tartışma başlıklarına bakın.
Hi. Do you want the design to be like this? Look at discussion titles.

![resim](https://github.com/community/community/assets/25375593/7d2c3293-057b-4223-a9b4-a33063faaf50)

**Comment by Talhadmr** [comment]
Volkan bey bu konuda  size katılıyorum. Github tartışma ksımını daha iyi tasarlamalı. Çok büyük bir potansiyel harcanıyor maalesef. Dğereli çalışmanız için de sizi tebrik ediyorum. Çalışmalarınız da başarılar diliyorum.

**Reply by volkantash** [reply]
Teşekkür ederim. Daha güzeli de yapılabilir. Ben yalnızca azıcık çaba ile biraz iyileştirme yaptım.
